src/main/kotlin/org/jboss/tools/intellij/kubernetes/model/util/KubeConfigContexts.kt

```diff
@@ -0,0 +1,127 @@
+/*******************************************************************************
```

Comment:
My concerns:

- I think this is a general concern utility so I would see it in intellij-common
- If the env var contains several files, my understanding is that aggregation is performed, I am wrong ? @joshuawilson  may help there

Reply:
> * I think this is a general concern utility so I would see it in intellij-common

This class is just a temporary solution. I want to contribute changes in kubernetes-client and think that this class wont be required any more. If there are parts which wont make it to the client library, I'll move it to intellij-common for sure.

Reply:
I just loaded 2 different kubeconfig files in 2 different locations. They did not merge. I can switch between them with `oc login [CLUSTER_URL]` but only one is active at a time.

Reply:
@jeffmaury, @joshuawilson: my code that is switching to a kubeconfig that's provided in the env var is copied from kubernetes-client. There's no functional change intended compared to what the client library provides. 
I don't see any merging happening but an override instead. If there's an env var that provides a kubeconfig it'll be used, if there's none, the default kubeconfig in the home directory is what's serverd.

Reply:
What we also discussed is that switching the project in an OpenShift cluster doesn't work. That's intended and is adressed in #27